### PR TITLE
[App service]:  az webapp config set : Add additional help examples for powershell users

### PR DESCRIPTION
**Pull request by StrawnSC**
**Description**

Adds additional examples to the help text for the command `az webapp config set` to help PowerShell users properly use the `--generic-configurations` option. PowerShell apparently requires more strict usage of escape characters than bash, so PowerShell users were having issues when trying to pass in JSON via this command. The new help text clarifies that PowerShell users must escape the double quotes in their JSON with a backslash (or pass in a JSON file name). 

**Testing Guide**

Generate the relevant help text with `az webapp config set --help`

**History Notes**

N/A

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
appservice

File: src/azure-cli/azure/cli/command_modules/appservice/_help.py
Comment (panchagnula) [inline src/azure-cli/azure/cli/command_modules/appservice/_help.py:1249]: we need to make the same updates for functional config set as well.

File: src/azure-cli/azure/cli/command_modules/appservice/_help.py
Comment (panchagnula) [inline src/azure-cli/azure/cli/command_modules/appservice/_help.py:1252]: @seligj95 can you review the text & provide the blob you'd like to use here in the help. Thanks!

**Review by panchagnula — COMMENTED** [review]
LGTM - but lets wait for text review from PM & incase he wants to update this.

File: src/azure-cli/azure/cli/command_modules/appservice/_help.py
Comment (seligj95) [inline src/azure-cli/azure/cli/command_modules/appservice/_help.py:1257]: I don't think it matters, but why are there 2 "\" in the path to the JSON? I just tried it and it works, but 1 "\" is enough.

**Review by seligj95 — COMMENTED** [review]
left a comment for review

**Review by seligj95 — APPROVED** [review]

**Pull request merged**
